# Translations template for ceilometer.
# Copyright (C) 2014 ORGANIZATION
# This file is distributed under the same license as the ceilometer project.
#
# Translators:
# Frédéric <frosmont@free.fr>, 2014
msgid ""
msgstr ""
"Project-Id-Version: Ceilometer\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2014-09-09 06:08+0000\n"
"PO-Revision-Date: 2014-09-02 00:36+0000\n"
"Last-Translator: openstackjenkins <jenkins@openstack.org>\n"
"Language-Team: French (http://www.transifex.com/projects/p/ceilometer/"
"language/fr/)\n"
"Language: fr\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"

#: ceilometer/collector.py:132
msgid "Dispatcher failed to handle the sample, requeue it."
msgstr ""

#: ceilometer/coordination.py:76
msgid "Error connecting to coordination backend."
msgstr ""

#: ceilometer/coordination.py:89
msgid "Error sending a heartbeat to coordination backend."
msgstr ""

#: ceilometer/coordination.py:142
msgid "Error getting group membership info from coordination backend."
msgstr ""

#: ceilometer/openstack/common/excutils.py:76
#, python-format
msgid "Original exception being dropped: %s"
msgstr "Exception d'origine en cours de suppression : %s"

#: ceilometer/openstack/common/excutils.py:105
#, python-format
msgid "Unexpected exception occurred %d time(s)... retrying."
msgstr "Exception inattendue survenue %d fois... Nouvel essai."

#: ceilometer/openstack/common/lockutils.py:117
#, python-format
msgid "Could not release the acquired lock `%s`"
msgstr "Impossible de libérer le verrou acquis `%s`"

#: ceilometer/openstack/common/loopingcall.py:95
msgid "in fixed duration looping call"
msgstr "dans l'appel en boucle de durée fixe"

#: ceilometer/openstack/common/loopingcall.py:138
msgid "in dynamic looping call"
msgstr "dans l'appel en boucle dynamique"

#: ceilometer/openstack/common/policy.py:508
#, python-format
msgid "Failed to understand rule %s"
msgstr "Règle %s incompréhensible"

#: ceilometer/openstack/common/policy.py:518
#, python-format
msgid "No handler for matches of kind %s"
msgstr "Aucun gestionnaire pour les correspondances de type %s"

#: ceilometer/openstack/common/policy.py:788
#, python-format
msgid "Failed to understand rule %r"
msgstr "Règle %r incompréhensible"

#: ceilometer/openstack/common/service.py:188
msgid "Exception during rpc cleanup."
msgstr "Exception pendant le nettoyage rpc."

#: ceilometer/openstack/common/service.py:277
msgid "Unhandled exception"
msgstr "Exception non gérée"
